### PR TITLE
Initialise Google Analytics only if we have CMP consents

### PR DESCRIPTION
**Pull request by QuarpT**
## Changes

Initialise google analytics only if we have the google-analytics CMP consent

## Why

We need consents from the user before we can start collecting Google Analytics

### CODE REVIEW

File: src/client/static/index.tsx [hunk 1]
```diff
@@ -15,13 +19,23 @@ import { init as ophanInit } from './analytics/ophan';
...
+    ) {
+      gaInit();
+    }
+  });
+};
+
 hydrateApp();
 
 // initalise ophan
 ophanInit();
```
Comment (coldlink) [inline src/client/static/index.tsx:36]: i know the ticket was only for ga, but do we want to consider something similar for ophan?

approving regardless

**Review by coldlink — APPROVED** [review]

File: src/client/static/index.tsx (hunk 1 above)
Comment (mchv) [reply src/client/static/index.tsx:36]: @coldlink We don't treat the ophan tracker and GA tracker in the same way for few different reasons:
-   `Ophan` is a `first-party` tracker while GA is `third-party` tracker. We can't send personal data to any third-party without disclosure, an updated privacy policy and a technical integration that protect the privacy of reader data _by design_.
- While we may intend to use `GA` with legitimate interest as lawful basis for processing, we found it too difficult to correctly configure `GA` to ensure that the data collected will be used only for that purpose. As such we only send data to GA for reader who consented for all purposes.
- For `Ophan` tracker, we have architected our data warehouse to ensure that data collected for legitimate interest purpose will never be integrated and joined with data collected for other purposes and  have put the right control in place.

We may use in the future different trackers to make this more seamless, but in the meantime the ophan tracker should not be removed as the data is collected and processed with legitimate interest as a lawful basis for processing.

Comment (mchv) [reply src/client/static/index.tsx:36]: @paulbrown1982 @dominickendrick are the right people to talk too about the use cases for the data collected by the ophan tracker under legitimate interest which is likely to include some customer journey analysis and optimisation.

Comment (coldlink) [reply src/client/static/index.tsx:36]: Very interesting to learn. Thanks @mchv 👍

**Pull request merged**